django/db/backends/sqlite3/operations.py

```diff
@@ -158,28 +161,38 @@ def quote_name(self, name):
     def no_limit_value(self):
         return -1
 
+    def __references_graph(self, table_name):
+        query = """
+        WITH tables AS (
+            SELECT %s name
+            UNION
+            SELECT sqlite_master.name
+            FROM sqlite_master
+            JOIN tables ON (
```

Comment:
`JOIN` can be single-lined `JOIN tables ON (sql REGEXP %s || tables.name || %s)`.